eng/SourceBuild.props

```diff
@@ -19,7 +19,7 @@
   <Target Name="CustomizeInnerBuildArgs"
           BeforeTargets="GetSourceBuildCommandConfiguration">
     <PropertyGroup>
-      <InnerBuildArgs>$(InnerBuildArgs) /p:SourceBuildRuntimeIdentifier=$(TargetRuntimeIdentifier)</InnerBuildArgs>
+      <InnerBuildArgs>$(InnerBuildArgs) /p:SourceBuildRuntimeIdentifier=$(TargetRuntimeIdentifier) /p:Projects=$(InnerSourceBuildRepoRoot)\Razor.SourceBuild.slnf</InnerBuildArgs>
```

Comment:
Can they build slnfs? I recall there being issues with it in the past.

Reply:
Yeah, I checked https://github.com/dotnet/source-build/issues/3161#issuecomment-1343206307